app/javascript/src/components/Common/Sidebar/constants.js

```diff
@@ -1,3 +1,4 @@
+import { UserCircle as Contacts } from "@bigbinary/neeto-icons";
 import { Notes, Settings } from "neetoicons";
```

Comment:
Why are there two different styles of import? Stick onto one of them. Preferably the latter one.

Reply:
This is was corrected afterwards.